dev/com.ibm.ws.clientcontainer.8.0_fat/build.gradle

```diff
@@ -0,0 +1,12 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
Is this okay that there's no EOL here?

Reply:
yeah think so - its def working because the fat test wouldn't work without it